src/Magento/FunctionalTestingFramework/Util/TestGenerator.php

```diff
@@ -759,7 +765,7 @@ public function generateStepsPhp($actionObjects, $generationScope = TestGenerato
                         $scope = PersistedObjectHandler::SUITE_SCOPE;
                     }
 
-                    $createEntityFunctionCall = "\t\tPersistedObjectHandler::getInstance()->createEntity(";
+                    $createEntityFunctionCall = "\t\t\${$actor}->createEntity(";
```

Comment:
For Allure cleanliness and since entity creation is now an `$I` function, we should do the following:
* Make the `createEntity` line be a one line long (to facilitate the next point)
* Print out stepKey at the end of the line like all other actions (for MagentoAllureAdapter to pick up)
* Remove the inclusion of a comment line before creating the Entity (see line 736)

This should also be done for all of the entity CRUD operations for cleanliness.

Reply:
Done